collector/topological_inventory.py

```diff
@@ -215,24 +220,32 @@ def worker(_: str, source_id: str, dest: str, acct_info: dict) -> None:
     headers = {"x-rh-identity": b64_identity}
 
     if ALL_TENANTS:
+        prometheus_metrics.METRICS['gets'].inc()
         resp = utils.retryable('get', TENANTS_URL, headers=headers)
+        prometheus_metrics.METRICS['get_successes'].inc()
         tenants_headers = \
             [tenant_header_info(t["external_tenant"]) for t in resp.json()]
 
-        LOGGER.info('Fetching data for ALL(%s) Tenants', len(tenants_headers))
+        LOGGER.info('Fetching data for ALL(%s) Tenants',
```

Comment:
I don't see what has been changed in this line?

Reply:
This line was broken down for pylama before because of the indentation applied earlier.

Since the indentation does not exist anymore, we can restore this line the way it was before.
Thanks!